jsonrpclib/SimpleJSONRPCServer.py

```diff
@@ -1,19 +1,26 @@
 import jsonrpclib
 from jsonrpclib import Fault
 from jsonrpclib.jsonrpc import USE_UNIX_SOCKETS
-import SimpleXMLRPCServer
-import SocketServer
+try:
+    import SimpleXMLRPCServer
+    import SocketServer
+except ImportError:
+    import xmlrpc.server as SimpleXMLRPCServer
+    import socketserver as SocketServer
 import socket
 import logging
 import os
-import types
 import traceback
 import sys
 try:
     import fcntl
 except ImportError:
     # For Windows
     fcntl = None
+try:
+    string_types = (str, unicode)
+except NameError:
+    string_types = (str, )
```

Comment:
Another option my be to simply do:
```python
try:
    basestring
except NameError:
    basestring = str
```

...and then use `isinstance(method, basestring)` since `basestring` is already defined for you in Python 2.7.